bindings/python/pfw.i

```diff
@@ -80,10 +80,18 @@ public:
 
     void setLogger(ILogger* pLogger);
 
-    ISelectionCriterionTypeInterface* createSelectionCriterionType(bool bIsInclusive);
-    ISelectionCriterionInterface* createSelectionCriterion(const std::string& strName,
-            const ISelectionCriterionTypeInterface* pSelectionCriterionType);
-    ISelectionCriterionInterface* getSelectionCriterion(const std::string& strName);
+    core::selection::criterion::CriterionInterface*
+    createExclusiveCriterion(const std::string& name,
+                             const core::selection::criterion::CriterionInterface::Values& values,
+                             std::string& error);
+
+    core::selection::criterion::CriterionInterface*
+    createInclusiveCriterion(const std::string& name,
+                             const core::selection::criterion::CriterionInterface::Values& values,
+                             std::string& error);
+
+    core::selection::criterion::CriterionInterface*
+    getSelectionCriterion(const std::string& name);
```

Comment:
please modify sample.py to take these changes into account. Maybe we should have a test for this.